contracts/contracts/test/ExampleDeployerListTest.sol

```diff
@@ -12,12 +12,14 @@ import "./AllowListTest.sol";
 // The transactions are put together as steps of a complete test in contract_deployer_allow_list.ts.
 // TODO: a bunch of these tests have repeated code that should be combined
 contract ExampleDeployerListTest is AllowListTest {
+  address constant ENABLED_ADDRESS = 0x0Fa8EA536Be85F32724D57A37758761B86416123;
+
   IAllowList allowList = IAllowList(DEPLOYER_LIST);
   ExampleDeployerList private example;
 
   function setUp() public {
     example = new ExampleDeployerList();
-    allowList.setNone(OTHER_ADDRESS);
+    allowList.setNone(ENABLED_ADDRESS);
```

Comment:
is this needed because the other test does not do a proper tear down?
(not needed to fix here, but could we open an issue or followup PR?)

Reply:
I presume, yes.

Reply:
Making sure the state is what you want it to be in `setUp` is better than relying on teardown. I agree that we should also do teardown, but that shouldn't change the `setUp` function here.